demos/browser/app/meetingV2/video/VideoTileCollection.ts

```diff
@@ -195,7 +195,12 @@ export default class VideoTileCollection implements AudioVideoObserver {
     }
     demoVideoTile.attendeeId = tileState.boundAttendeeId;
 
-    demoVideoTile.show(tileState.isContent);
+    if (tileState.active || tileState.paused) {
+        demoVideoTile.show(tileState.isContent);
+    } else {
+        // Hide non-active tiles that aren't just paused
+        demoVideoTile.hide();
```

Comment:
When does this happen?

Reply:
This is to fix an unrelated issue in that local tile will not be hidden when toggled off (hence the active check). However that would also hide paused tiles (no unpausing!), hence the pause check.